BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -152,6 +152,97 @@ public static Vector3 GetAirToAirTarget(Vector3 targetPosition, Vector3 targetVe
+                ttgoWeave = ttgo;
+            }
+            else
+            {
+                Vector3 weaveDir = (targetPosition - VectorUtils.GetWorldSurfacePostion(weaveStart, missileVessel.mainBody)).ProjectOnPlanePreNormalized(upDirection).normalized;
+                ttgoWeave = weaveFactor * 1.5f * Vector3.Dot(Rdir, weaveDir) / speed;
+                right = Vector3.Cross(weaveDir, upDirection);
+            }
+                    
+            float omegaBeta = PI2 * omega * ttgoWeave;
+            float sinOmegaBeta = Mathf.Sin(omegaBeta);
+            float cosOmegaBeta = Mathf.Cos(omegaBeta);
+
+            float sinOmegaBetaOff = Mathf.Sin(omegaBeta - weaveOffset);
+            float cosOmegaBetaOff = Mathf.Cos(omegaBeta - weaveOffset);
+
+            const float g = 9.80665f;
+            float ka = 2f * omegaBeta * sinOmegaBeta + 6f * cosOmegaBeta - 6f;
+            float kj = -2f * omegaBeta * cosOmegaBeta + 6f * sinOmegaBeta - 4f * omegaBeta;
+
+            float ttgoWeaveInv = 1f / ttgoWeave;
+            float omegaBetaInv = 1f / (Mathf.Max(omegaBeta * omegaBeta, 0.000001f));
+
+            float aVert = ((terminalAngle > 0) ? (speed * (6f * Mathf.Asin(Vector3.Dot(upDirection, Rdir) / Rdir.magnitude) - 4f * verticalAngle + 2f * terminalAngle * Mathf.Deg2Rad) * ttgoWeaveInv) : 0.0f) // A_BPN                                                                                                                                    
+                + ((gVert != 0.0f) ? (gVert * g * ((ka + omegaBeta * omegaBeta) * sinOmegaBetaOff + kj * cosOmegaBetaOff) * omegaBetaInv) : 0.0f); // A_W
+            float aHor = ((terminalAngle > 0) ? (-6f * speed * horizontalAngle) * ttgoWeaveInv : 0.0f) // A_BPN            
+                + ((gHorz != 0.0f) ? (gHorz * g * ((ka + omegaBeta * omegaBeta) * cosOmegaBetaOff + kj * sinOmegaBetaOff) * omegaBetaInv) : 0.0f); // A_W
+
+            if (BDArmorySettings.DEBUG_MISSILES)
+                Debug.Log($"[BDArmory.MissileGuidance] Weave guidance ttgoWeave: {ttgoWeave}, omegaBeta: {omegaBeta}, ka: {ka}, kj: {kj}, vertAngle: {Mathf.Rad2Deg * verticalAngle}, horAngle: {Mathf.Rad2Deg * horizontalAngle}, aVert: {aVert} m/s^2, aHor: {aHor} m/s^2.");
+
+            Quaternion rotationPitch = Quaternion.AngleAxis(verticalAngle, right);
+            Quaternion rotationYaw = Quaternion.AngleAxis(horizontalAngle, upDirection);
+
+            Vector3 accel = (aVert * (rotationPitch * rotationYaw * upDirection) + aHor * (rotationYaw * right));// + GetPNAccel(targetPosition, targetVelocity, missileVessel, 3f);
+            if (terminalAngle < 0)
+            {
+                accel = accel + GetPNAccel(targetPosition, targetVelocity, missileVessel, 3f);
+            }
+
+            gLimit = accel.magnitude;
+
+            float leadTime = Mathf.Min(4f, ttgoWeave);
+
+            return missileVessel.CoM + leadTime * missileVel + accel * (leadTime * leadTime);
```

Comment:
The acceleration is missing a factor of 1/2, cf.,
```csharp
AIUtils.PredictPosition(missileVessel.CoM, missileVel, accel, leadTime);
```

Reply:
Woops, don't know how that slipped my mind